Steer/src/MCKinematicsReader.cxx

```diff
@@ -175,3 +175,33 @@ bool MCKinematicsReader::initFromKinematics(std::string_view name)
 
   return true;
 }
+
+bool MCKinematicsReader::initFromKinematics(std::vector<std::string> names)
```

Comment:
is this new function really needed?

Reply:
Hi @sawenzel

If I understand correctly, in case when digitization context is not used it is not possible to load events from multiple sources using this [MCKinematicsReader::initFromKinematics(std::string_view name)](https://github.com/AliceO2Group/AliceO2/blob/720c1efc4713578d2835242318a365cd4ed1ef5e/Steer/src/MCKinematicsReader.cxx#L163-L177 ). To initialize MC reader one needs to access private fields, so I introduced an overloaded function.

Reply:
In my opinion loading from multiple sources does not make so much sense when not using collisioncontext.

Reply:
Thanks!